Client.Linq/Internal/Expressions/String/Replace.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class Replace : IExpressionPart
+    {
+        private readonly IEnumerable<IExpressionPart> expressionParts;
+        private readonly IEnumerable<IExpressionPart> expressionParts2;
+        private readonly IEnumerable<IExpressionPart> expressionParts3;
+
+        internal Replace(IEnumerable<IExpressionPart> expressionParts, IEnumerable<IExpressionPart> expressionParts2,
+            IEnumerable<IExpressionPart> expressionParts3)
+        {
+            this.expressionParts = expressionParts;
+            this.expressionParts2 = expressionParts2;
+            this.expressionParts3 = expressionParts3;
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("strings.replaceAll(v: ");
+            foreach (var expressionPart in expressionParts) expressionPart.AppendFlux(builder);
+            builder.Append(", t: ");
+            foreach (var expressionPart in expressionParts2) expressionPart.AppendFlux(builder);
+            builder.Append(", u: ");
+            foreach (var expressionPart in expressionParts3) expressionPart.AppendFlux(builder);
+            builder.Append(")");
+        }
```

Comment:
Please use same naming as is in Flux documentation:

```suggestion
        private readonly IEnumerable<IExpressionPart> _value;
        private readonly IEnumerable<IExpressionPart> _substring;
        private readonly IEnumerable<IExpressionPart> _replacement;

        internal Replace(IEnumerable<IExpressionPart> value, IEnumerable<IExpressionPart> substring,
            IEnumerable<IExpressionPart> replacement)
        {
            _value = value;
            _substring = substring;
            _replacement = replacement;
        }

        public void AppendFlux(StringBuilder builder)
        {
            builder.Append("strings.replaceAll(v: ");
            foreach (var expressionPart in _value) expressionPart.AppendFlux(builder);
            builder.Append(", t: ");
            foreach (var expressionPart in _substring) expressionPart.AppendFlux(builder);
            builder.Append(", u: ");
            foreach (var expressionPart in _replacement) expressionPart.AppendFlux(builder);
            builder.Append(")");
        }
```